mne/epochs.py

```diff
@@ -1233,6 +1242,16 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
+                if any([k_e not in list(epochs.event_id.keys())
```

Comment:
you do not need to do `x in list(my_dict.keys())` you can just do `x in my_dict` directly